2200005/1204.py

```diff
@@ -0,0 +1,16 @@
+a= int(input())
+for i in range(a):
+    num = input()
+    d = {}
+    l = list(map(int,input().split()))
+    for j in l:
+        d[j] = d.get(j,0) +1
+    l2 = sorted(d.items(),key=lambda x:-x[1])
+    l3= []
+    for k in l2:
+        l3.append(l2[0][0])
+        if k[1] == l2[0][1]:
+            l3.append(k[0])
+        else:
+            break
+        print(f'#{i+1} {max(l3)}')
```

Comment:
좋은 풀이네요! 😄 
```suggestion
    print(f'#{i+1} {max(l3)}')
```
Python은 여백이 큰 의미를 가지므로, 신경 써 주시면 더욱 좋을 것 같습니다 👍